Dockerfile

```diff
@@ -138,13 +151,15 @@ ENV HZ_CLIENT_MODE=false \
 RUN  mkdir -p /opt/pega/kafkadata && \
      chgrp -R 0 /opt/pega/kafkadata && \
```

Comment:
Again, we're changing group to root group, but it seems like it should already be root group.

Reply:
same as heapdump